packages/block-library/src/navigation/style.scss

```diff
@@ -208,35 +208,22 @@
 
 /**
  * Margins
- * @todo: refactor this to use gap.
  */
 
 // Menu items with no background.
-.wp-block-page-list,
-.wp-block-page-list > .wp-block-navigation-item,
-.wp-block-navigation__container > .wp-block-navigation-item {
-	margin: 0 2em 0 0;
-
-	// Margin of right-most margin should be zero, for right aligned or justified items.
-	&:last-child {
-		margin-right: 0;
-	}
+.wp-block-navigation__container .wp-block-page-list,
```

Comment:
On the front end, `wp-block-page-list` isn't inside `wp-block-navigation__container`, so this won't apply. I think we could go with `.wp-block-navigation .wp-block-page-list` instead.

Reply:
Excellent, that worked:

<img width="770" alt="Screenshot 2021-09-01 at 10 20 27" src="https://user-images.githubusercontent.com/1204802/131637704-49e50e4d-51f8-4d83-94f2-1b53b11c5a92.png">

<img width="768" alt="Screenshot 2021-09-01 at 10 20 36" src="https://user-images.githubusercontent.com/1204802/131637711-b563c3c2-1dc8-4e4c-a949-caa67b916c3a.png">